src/test/groups.test.js

```diff
@@ -73,5 +73,24 @@ describe("Groups", function () {
 
       GroupsAction.deleteGroup("/group-non-existent");
     });
+
+    it("emits an error on a non-existing group with statuscode 404",
```

Comment:
Couldn't we simply add the `statusCode` check to the already existing test at line 60?